edx_sysadmin/static/edx_sysadmin/css/style.css

```diff
@@ -56,6 +56,17 @@ table.stat_table td {
     background-color: #ffffff;
 }
 
+table.stat_table.courses_table {
+    width: 100%;
+}
+
+table.stat_table.courses_table td button {
+    display: block;
+    margin: auto;
+    width: 170px;
+    height: 60px;
+}
```

Comment:
Is there any existing button style we can re-use here? If not, this is fine.

Reply:
We are using edX's basic button styles throughout edx-sysadmin, the current button is also using the same style. The little tweaks which I added are just to `center align the button in the table column` and secondly to `show same-sized buttons when text changes from **Load Details** to **Update Details**`.